src/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -122,9 +122,9 @@ msgstr ""
 
 #: services/messenger.py:236
 msgid ""
-"According to the dictionary, Open-source 🔓 software, denotes software for"
-" which the original source code is made freely 🆓 available and may be "
-"redistributed and modified."
+"Open-Source Software, is any Software available for modification ⌨️."
```

Comment:
Is this an authoritative definition? It seems to belong to Patricia Picket: https://www.thebalancecareers.com/what-is-open-source-software-2071941

A much formal definition can be  **Open source software is software with source code that anyone can inspect, modify, and enhance.** : https://opensource.com/resources/what-open-source , which for me is quite similar to the version you want to modify!

Reply:
Yeah. This definition is similar to the one in Patricia Picket article.
I feel the Opensource.com definition sounds better. Do you want to update it, or should I update it?